wdl-lint/src/rules/import_whitespace.rs

```diff
@@ -27,7 +27,7 @@ fn blank_between_imports(span: Span) -> Diagnostic {
     Diagnostic::note("blank lines are not allowed between imports")
         .with_rule(ID)
         .with_highlight(span)
-        .with_fix("remove any blank lines between imports")
+        .with_fix("remove the blank line between imports")
```

Comment:
```suggestion
        .with_fix("remove blank lines between imports")
```

We probably still need to handle cases where there are multiple lines, right?